Error compiling smc-command in OSX 10.9.1

I'm having problems compiling in my MBA 2013.
Here's the problem:

```
pedro@Pedros-MacBook-Air-2:~/smcFanControl/smc-command$ make
gcc -c smc.c
smc.c:384:23: warning: format string is not a string literal (potentially insecure) [-Wformat-security]
    sprintf(val->key, key);
                      ^~~
/usr/include/secure/_stdio.h:47:56: note: expanded from macro 'sprintf'
  __builtin___sprintf_chk (str, 0, __darwin_obsz(str), __VA_ARGS__)
                                                       ^
1 warning generated.
gcc -mmacosx-version-min=10.4  -Wall -g -framework IOKit -o smc smc.o
Undefined symbols for architecture x86_64:
  "_main", referenced from:
      __start in crt1.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make: *** [smc] Error 1
```

Any ideas?

Thanks

--- a/smc-command/smc.h
+++ b/smc-command/smc.h
@@ -22,6 +22,7 @@
 #define __SMC_H__
 #endif
 
+#define CMD_TOOL
 #define VERSION               "0.01"
 
 #define OP_NONE               0

